app/controllers/groups_controller.rb

```diff
@@ -8,32 +8,20 @@
 class GroupsController < ApplicationController
   self.permitted_attrs = [:name, :description]
 
+  before_action :group, except: [:new, :create]
```

Comment:
is this needed ?
the idea of the group getter is to cache it as soon as it's required. 
so if you access group by group and not by @group, this should work perfectly